backend/index.php

```diff
@@ -13,11 +13,11 @@
 
 $container = require __DIR__ . "/Application/AppContainerBuilder.php";
 $application = $container->get(Application::class);
-$options = getopt("rf");
+$options = getopt("", ["populate", "fetch"]);
```

Comment:
Can we use named arguments and get rid this first param?

Reply:
It seems I cannot do that. First parameter is required. 
I can use named arguments.

Reply:
Ah, ok, ok